meson.build

```diff
@@ -529,10 +530,23 @@ if have_gumjs
     quickjs_dep_native = dependency('quickjs', native: true)
   endif
 
-  v8_dep = dependency('v8-10.0',
-    version: '>=10.6.122',
-    required: get_option('v8'),
+  # workaround for https://github.com/mesonbuild/meson/issues/11381
+  if get_option('v8').auto()
+    error('option "v8" cannot be "auto", must be either "enabled" or "disabled"')
+  endif
```

Comment:
I take it that this is an outdated solution?

Reply:
fixed